lib/uinta/plug.ex

```diff
@@ -110,7 +110,20 @@ if Code.ensure_loaded?(Plug) do
 
     @impl Plug
     def init(opts) do
-      format = if Keyword.get(opts, :json, false), do: :json, else: :string
+      format =
+        case Keyword.fetch(opts, :format) do
+          {:ok, :json = value} ->
+            value
+
+          {:ok, :map = value} ->
+            value
+
+          {:ok, :string = value} ->
+            value
+
+          :error ->
+            if Keyword.get(opts, :json, false), do: :json, else: :string
```

Comment:
That case statement seems a little complex. Why not make the list of options a MapSet that is compiled (@valid_formats). and check if this is a member? that would simplify the code

Reply:
@epinault I changed the check to be more compact.
This should not be a breaking change, per se, more of a deprecation, as the old `json` option still works.